test/module/irohad/ametsuchi/redis_block_query_test.cpp

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ametsuchi/impl/redis_flat_block_query.hpp"
+#include "ametsuchi/impl/storage_impl.hpp"
+#include "crypto/hash.hpp"
+#include "framework/test_subscriber.hpp"
+#include "module/irohad/ametsuchi/ametsuchi_fixture.hpp"
+
+using namespace iroha::ametsuchi;
+using namespace iroha::model;
+using namespace framework::test_subscriber;
+
+/**
+ * @given block store with 2 blocks totally containing 3 txs created by
+ * user1@test and 1 tx created by user2@test
+ * @when queries to get transactions created by both users are invoked
+ * @then query over user1@test returns 3 txs and query over user2@test returns 1
+ * tx and query over non-existing user returns 0 txs
+ */
+TEST_F(AmetsuchiTest, GetAccountTransactions) {
+  auto storage =
```

Comment:
Since the test is for `RedisFlatBlockQuery`, it does not seem right that it uses `Storage` and `MutableStorage`. It is possible to rewrite the test using only `RedisFlatBlockQuery` with its dependencies (`redis_client` and `FlatFile`)?

Reply:
No reasons to rewrite test with this manner, better to add new unit test instead.

Reply:
How to put test data without using mutable storage?

Reply:
Just create index with fake data